app/app.go

```diff
@@ -290,6 +293,10 @@ type App struct {
 	EpochKeeper epochmodulekeeper.Keeper
 
 	TokenFactoryKeeper tokenfactorykeeper.Keeper
+
+	// Ethermint keepers
+	EvmKeeper       *evmkeeper.Keeper
+	FeeMarketKeeper feemarketkeeper.Keeper
```

Comment:
how does the feemarket module interact with the original gas fee system here?

Reply:
This is for allowing for variable fee for Cosmos txns so that it behaves similarly to EIP-1559: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md
Do you think it makes sense to remove it and keep the original gas fee mechanism? \cc @codchen

Reply:
we can discuss this in a separate ticket^